README.md

```diff
@@ -58,7 +58,7 @@ bash Miniforge3-Linux-aarch64.sh -bfp ~/conda
 rm Miniforge3-Linux-aarch64.sh
 source ~/conda/etc/profile.d/conda.sh
 conda activate
-conda install -y -c maddenp condev=0.4.0
+conda install -y -c maddenp condev=0.5.3
```

Comment:
Here and in the diff just below, update `condev` versions to track updates to that package. Older versions will generally work, but when I can verify that the latest release works, I'd like to suggest using that.